client/src/components/dashboard/Community/UserCard.js

```diff
@@ -71,8 +91,10 @@ class UserCard extends React.Component {
 
     const {
       user,
+      user: {
```

Comment:
@smiller171 one is the entire user object which we need for certain parts of the function, and to access other parts of the object that aren't repeated, however `username` is repeated often enough where pulling that out separately was worth it. I'm not sure what you mean by wrapping it in quotes - when destructuring this object, you would get a syntax error if you wrote it like this:
```
const {
      user,
      user: {
        'username
      },
     // ...
  } = this.props;
```
I believe this is ok as it is.